services/api/src/smtp_error_handler.py

```diff
@@ -10,28 +10,10 @@
 import ssl
 import os
 
-import common.pgquery as pgquery
-
 
 def get_subscribed_users():
-    query = "SELECT email FROM public.users WHERE receive_error_emails = '1'"
-
-    data = pgquery.query_db(query)
-
-    return [point["email"] for point in data]
-
-
-def unsubscribe_user(email: str):
-    pre_check = f"SELECT receive_error_emails FROM public.users WHERE email = '{email}'"
-    pre_check_results = pgquery.query_db(pre_check)
-    if not pre_check_results:
-        return 400
-
-    query = f"UPDATE public.users SET receive_error_emails='0' WHERE email = '{email}'"
-
-    pgquery.write_db(query)
-
-    return 200
+    emails = os.environ["CSM_EMAILS_TO_SEND_TO"].split(",")
```

Comment:
Should this env var still be getting used?

Reply:
Same reason applies here.